File: app/src/main/java/com/github/premnirmal/ticker/network/GoogleFinance.java
package com.github.premnirmal.ticker.network;

import java.util.List;

import retrofit.http.GET;
import rx.Observable;

/**
 * Created by premnirmal on 3/15/15.
 */
public interface GoogleFinance {

    @GET("/info")
    Observable<List<GStock>> getStock(@retrofit.http.Query(value = "q", encodeValue = false) String query);
}

File: app/src/main/kotlin/com/github/premnirmal/ticker/network/GoogleFinance.kt
package com.github.premnirmal.ticker.network

import com.github.premnirmal.ticker.network.data.GStock
import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable

/**
 * Created by premnirmal on 3/3/16.
 */
interface GoogleFinance {

  @GET("info")
  fun getStocks(@Query(value = "q", encoded = false) query: String): Observable<List<GStock>>
}